### PR TITLE
Hierarchy sum process root nodes

### PR DESCRIPTION
**Pull request by youlot**

### CODE REVIEW

File: src/app/js/formats/streamgraph/Streamgraph.js [hunk 1]
```diff
@@ -101,7 +101,8 @@ class Streamgraph extends PureComponent {
     initTheGraphBasicsElements(width, height, margin, svgViewport) {
         const zoom = d3
             .zoom()
-            .scaleExtent([1, 32])
+            //.scaleExtent([1, 32])
+            .scaleExtent([1, 572])
```
Comment (nojhamster) [inline src/app/js/formats/streamgraph/Streamgraph.js]: Please resolve conflict with #874 before merging

**Review by nojhamster — CHANGES_REQUESTED** [review]

File: src/app/js/formats/streamgraph/Streamgraph.js (hunk 1 above)
Comment (youlot) [reply src/app/js/formats/streamgraph/Streamgraph.js]: that's done :)

**Review by nojhamster — CHANGES_REQUESTED** [review]
Seems another PR created a conflict

**Comment by youlot** [comment]
> Seems another PR created a conflict

I resolved it :)

**Pull request merged**